include/courtroom.h

```diff
@@ -603,6 +607,7 @@ private slots:
   void on_objection_clicked();
   void on_take_that_clicked();
   void on_custom_objection_clicked();
+  void ShowContextMenu(const QPoint& pos);
```

Comment:
```suggestion
  void show_context_menu(const QPoint& pos);
```

Reply:
This is a qt function so I can't change the name

Reply:
No. Remember that you `connect` it to the function name you want.